content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -41,7 +42,7 @@ The following methods exist for installing kubectl on Windows:
    Download the `kubectl` checksum file:
 
    ```powershell
-   curl -LO "https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe.sha256"
+   Invoke-WebRequest -Uri "https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe.sha256" -OutFile "kubectl.exe.sha256"
```

Comment:
```suggestion
   curl.exe -LO "https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe.sha256"
```

Reply:
I think Invoke-Webrequest would be more conventient for native windows users since powershell is the goto command line for most power  users.